tools/packaging/postinstall.sh

```diff
@@ -11,12 +11,15 @@ fi
 cleanInstall() {
     printf "\033[32m Post Install of a clean install\033[0m\n"
 
+    if ! getent group pyroscope >/dev/null 2>&1; then
+        groupadd -r pyroscope
```

Comment:
Long arguments are always nicer to understand:

```suggestion
        groupadd --system pyroscope
```